### DISCUSSION TITLE
How to get the  default threshold？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#184] [Q&A]
How to calculate the  default threshold？Below is in the example of function “imbalanced.rfsrc”
![image](https://user-images.githubusercontent.com/5112397/149687186-011fe07f-1c46-48a2-9dd9-99050a35601e.png)

**Comment by ishwaran** [comment]
This is described in the following vignette, but essentially the default threshold is the observed relative frequency of the minority class label

https://luminwin.github.io/randomForestSRC/articles/imbalance.html
